tests/test_win_prob.py

```diff
@@ -0,0 +1,49 @@
+"""Tests Win Probability module."""
+
+import pytest
+
+from awpy import Demo
+from awpy.stats import win_probability
+
+class TestWinProbability:
+    """Tests the win probability calculations.
+    
+    https://www.hltv.org/matches/2369248/natus-vincere-vs-virtuspro-pgl-cs2-major-copenhagen-2024-europe-rmr-closed-qualifier-a
+    """
+
+    @classmethod
+    def setup_class(cls):
+        """
+        Setup method called before any tests are run.
+        Initializes the Demo object and random weights for testing.
+        """
+        cls.demo = Demo(file='tests/natus-vincere-vs-virtus-pro-m1-overpass.dem')  
+
+
+    @classmethod
+    def teardown_class(cls):
+        """
+        Teardown method called after all tests have run.
+        """
+        cls.demo = None
+
+
+    def test_win_probability_symmetry(self):
+        """Test to ensure P(CT Win) = 1 - P(T Win) at a given tick."""
+        probabilities = win_probability(self.demo, 168200)
+        for prob in probabilities:
+            assert prob["CT_win_probability"] == pytest.approx(1 - prob["T_win_probability"], 0.01)
```

Comment:
I would probably assert that CT win prob + T win prob = 1.0

We might also want a stronger tolerance than 0.01. Probably the default is fine (so just `pytest.approx(1.0)`)

Reply:
cleaned up code and now use a stronger tolerance.